lib/kitchen/driver/openstack.rb

```diff
@@ -324,12 +321,26 @@ def parse_ips(pub, priv)
       end
 
       def add_ohai_hint(state)
-        info 'Adding OpenStack hint for ohai'
-        ssh = Fog::SSH.new(*build_ssh_args(state))
-        ssh.run([
-          %(sudo mkdir -p #{Ohai::Config[:hints_path][0]}),
-          %(sudo touch #{Ohai::Config[:hints_path][0]}/openstack.json)
-        ])
+        if bourne_shell?
+          info 'Adding OpenStack hint for ohai'
+          mkdir_cmd = "sudo mkdir -p #{hints_path}"
+          touch_cmd = "sudo touch #{hints_path}/openstack.json"
+          instance.transport.connection(state).execute(
+            "#{mkdir_cmd} && #{touch_cmd}"
+          )
+        end
+        if windows_os?
```

Comment:
nitpick, but this feels like it's part of the same if block as above, and perhaps the `end` above it should be removed and this becomes `elsif windows_os?`